apps/app/dashboards/demography/refugee-situation/index.tsx

```diff
@@ -0,0 +1,283 @@
+import BarMeter from "@components/Chart/BarMeter";
+import Slider from "@components/Chart/Slider";
+import { SliderProvider } from "@components/Chart/Slider/context";
+import { UNHCRIcon } from "@components/Icon/agency";
+import {
+  AgencyBadge,
+  Container,
+  Dropdown,
+  Hero,
+  LeftRightCard,
+  Section,
+  Tooltip,
+} from "@components/index";
+import { OptionType } from "@components/types";
+import { useData } from "@hooks/useData";
+import { useSlice } from "@hooks/useSlice";
+import { useTranslation } from "@hooks/useTranslation";
+import { AKSARA_COLOR, CountryAndStates } from "@lib/constants";
+import { getTopIndices, numFormat, toDate } from "@lib/helpers";
+import dynamic from "next/dynamic";
+import { FunctionComponent } from "react";
+
+/**
+ * Refugee Situation Dashboard
+ * @overview Status: In-development
+ */
+
+const Choropleth = dynamic(() => import("@components/Chart/Choropleth"), { ssr: false });
+const Timeseries = dynamic(() => import("@components/Chart/Timeseries"), { ssr: false });
+
+interface RefugeeSituationProps {
+  barmeter: any;
+  choropleth: any;
+  last_updated: any;
+  timeseries: any;
+  timeseries_callout: any;
+}
+
+const RefugeeSituation: FunctionComponent<RefugeeSituationProps> = ({
+  barmeter,
+  choropleth,
+  last_updated,
+  timeseries,
+  timeseries_callout,
+}) => {
+  const { t, i18n } = useTranslation(["dashboard-refugee-situation", "common"]);
+  const FILTER_OPTIONS: Array<OptionType> = ["absolute", "per_capita", "perc"].map(
+    (key: string) => ({
+      label: t(`${key}`),
+      value: key,
+    })
+  );
+  const { data, setData } = useData({
+    tab_index: 0,
+    timeseries: timeseries,
+    timeseries_callout: timeseries_callout,
```

Comment:
if the data for timeseries and callouts are unchanging, you can just rely on them as `props`. Dont need to reassign them as a state.